scenarios/kubernetes_e2e.py

```diff
@@ -159,28 +163,28 @@ def main(args):
         '--gce-ssh',
         default=os.environ.get('JENKINS_GCE_SSH_PRIVATE_KEY_FILE'),
         help='Path to .ssh/google_compute_engine keys')
-
     PARSER.add_argument(
         '--gce-pub',
         default=os.environ.get('JENKINS_GCE_SSH_PUBLIC_KEY_FILE'),
         help='Path to pub gce ssh key')
-
     PARSER.add_argument(
         '--service-account',
         default=os.environ.get('GOOGLE_APPLICATION_CREDENTIALS'),
         help='Path to service-account.json')
 
     # Assume we're upping, testing, and downing a cluster by default
     PARSER.add_argument(
-        '--up', default='true', help='If we need to set --up in e2e.go')
+        '--up', action='store_true', default='true', help='If we need to set --up in e2e.go')
```

Comment:
Will you revert this change? There is no way to set --up=false now, and --up will actually cause it not to up the cluster because `True != true`

That's why I suggested making this change in another PR.

You should either refactor this to be something like `--skip-up` or else follow the `nargs='?'` pattern that bootstrap uses (if you want to support the following: --up, --up=1, --up=0) -- https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L808

Also please be sure to handle the fact that args.up is now a boolean, not a string, thus this arg is no longer correct:

` '-e', 'E2E_UP=%s' % args.up,`

We need this to be 'true' when args.up, which is only the case right now because you're setting the default value to the string 'true' rather than the boolean True. Note that typically you should not set defaults for store_true actions